Fix for issue #165: add doc strings to launch file args

gavanderhoorn opened the pull request:
As per subject.

No functional changes, only adds documentation. Support for this was added to `roslaunch` in `v1.11.1`.

VictorLamoine commented on fanuc_driver/launch/motion_streaming_interface.launch:
```diff
@@ -3,17 +3,17 @@
 
   <!-- Load the byte-swapping version of robot_state if required -->
-  <arg name="use_bswap" />
+  <arg name="use_bswap" doc="Iff true, driver will perform byte-swapping on incoming data stream" />
```
Typo on `Iff`

gavanderhoorn replied:
This is the mathematical / logical [if and only if](https://en.wikipedia.org/wiki/If_and_only_if). So not a typo, more a habit ("only if this is set to true, will ..").

gavanderhoorn commented:
@JeremyZoss: could I ask you to take a look again?

JeremyZoss commented:
Looks good.  A few potential changes (not required):
- for `robot_ip`, note that it's not required if `sim=true` (will show up as "required arg" in new `--ros-args`)
- I think `Iff` will generate more questions than needed.  It doesn't really do much to clarify here.
- Clarify wording on `use_bswap`.  Technically, bswap on both incoming/outgoing data streams, but only the data streams to robot (not ROS side).  Perhaps something like:
  - `If true, driver will byte-swap all data to/from the robot controller`

Overall, looks fine.  +1 from me.  This is a welcome feature, and a good catch to add (hasn't even made the official wiki docs yet!)

gavanderhoorn commented:
@JeremyZoss: I added a note about `robot_ip` and `use_bswap` being only required when `sim:=false`, updated doc string for `use_bswap` and removed the _iff_. As I wrote earlier, a habit I guess.

@VictorLamoine: thanks for taking a look at the PR.

gavanderhoorn commented:
I'll go ahead and merge this.
